app/settings.py

```diff
@@ -266,3 +267,5 @@
 MAX_VOTES = 10
 
 MAX_VOTES_TO_APP = 50
+
+SPONSOR_TOKEN = os.environ.get('SPONSOR_TOKEN', None)
```

Comment:
Why sponsor token? Can we not avoid having to send sponsors a token by generating a single URL for each sponsor instead?

Reply:
mb, this is old. must be deleted